src/common/components/select-repository-list/index.js

```diff
@@ -63,20 +65,22 @@ export class SelectRepositoryListComponent extends Component {
   }
 
   renderRepository(repo) {
-    const { fullName, enabled } = repo;
+    const { fullName } = repo;
     const status = this.props.repositoriesStatus[fullName] || {};
-    const { selectedRepos } = this.props.selectRepositoriesForm;
+
+    const selected = this.props.isRepoSelected(repo);
+    const enabled = this.props.isRepoEnabled(repo);
 
     return (
       <SelectRepositoryRow
         key={ `repo_${repo.fullName}` }
         repository={ repo }
-        buttonLabel={ status.isFetching ? 'Creating...' : 'Create' }
-        buttonDisabled={ status.isFetching }
         onChange={ this.onSelectRepository.bind(this, repo) }
         errorMsg= { this.getErrorMessage(status.error) }
-        checked={ hasRepository(selectedRepos, repo) }
-        isEnabled={ enabled }
```

Comment:
It was confusing to have 2 properties (`checked` and `isEnabled`) that would define is checkbox is checked.